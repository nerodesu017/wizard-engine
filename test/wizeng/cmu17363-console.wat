(module
  (import "console" "log_int" (func $log_int (param i32)))
  (import "console" "log_bool" (func $log_bool (param i32)))
  (import "js" "mem" (memory 0))
  
  (func $main (export "main")
    (call $log_int (i32.const 11223344))
    (call $log_int (i32.const -66778899))
    (call $log_int (memory.size))
    (call $log_bool (i32.const 1))
    (call $log_bool (i32.const 0))
  )
)
