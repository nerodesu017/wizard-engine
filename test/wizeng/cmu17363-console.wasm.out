11223344
-66778899
true
false
